src/asciidoctor-extensions-api.js

```diff
@@ -234,3 +245,10 @@ if (typeof Extensions.TreeProcessor !== 'undefined') {
 TreeProcessor.$$proto.process = function (block) {
   return Opal.send(this, 'process', null, toBlock(block));
 };
+
+/**
+ * @namespace
+ * @module Extensions/Preprocessor
+ */
+// eslint-disable-next-line no-unused-vars
+var Preprocessor = Extensions.Preprocessor;
```

Comment:
I don't think this line is needed. There are no special methods on the Preprocessor class.